build-server.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
let's get rid of this (and the corresponding gitignore) for now. the cli will do the build step for you so you shouldn't need to do this manually

Reply:
Sounds good